data/variants.txt

```diff
@@ -575,13 +577,15 @@ ship "Cruiser" "Cruiser (Heavy)"
 		"Quad Blaster Turret"
 		"Laser Rifle" 20
 		"Fragmentation Grenades" 20
+		"Brig"
 		"Armageddon Core"
 		"Dwarf Core"
 		"Supercapacitor"
 		"D94-YV Shield Generator"
 		"Liquid Helium Cooler"
 		"Ramscoop"
 		"Outfits Expansion"
+		"Brig"
```

Comment:
I assume "Brig" isn't supposed to be listed twice for this ship.

Reply:
Fixed.